src/dotnet/Program.cs

```diff
@@ -78,14 +76,15 @@ public static int Main(string[] args)
             }
         }
 
-        internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
+        internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null)
```

Comment:
Why did we make this change? Where else is this ProcessArgs being used where a telemetryClient is used?